api/src/types/Transaction.ts

```diff
@@ -25,7 +25,7 @@ export class GearTransaction {
   public signAndSend(
     account: AddressOrPair,
     optionsOrCallback?: Partial<SignerOptions> | TransactionStatusCb,
-    optionalCallback?: (data: any) => void
+    optionalCallback?: (data: any) => void,
```

Comment:
You could use `unknown` instead of `any` here

Reply:
same